raven/processes/wps_generic_zonal_stats.py

```diff
@@ -51,7 +49,7 @@ def __init__(self):
                          data_type='boolean', default='false',
                          min_occurs=1, max_occurs=1),
             LiteralInput('WCS_VERSION', 'DEBUG FIELD SPECIFYING WCS VERSION USED TO ACCESS DEM',
-                         data_type='string', default='1.0.0',
+                         data_type='string', default='1.0.0', allowed_values=['1.0.0', '2.0.1'],
```

Comment:
Should we remove this now ?

Reply:
If we want to stick with WCS 2.0.1, we no longer need this or the CRS definition in the `get_dem` function. Want me to clean that up?

Reply:
Yes